app/Jobs/UpdateMajorsForUser.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Jobs;
+
+use App\User;
+use Exception;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use OITNetworkServices\BuzzAPI;
+use OITNetworkServices\BuzzAPI\Resources;
+
+class UpdateMajorsForUser implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    /**
+     * The number of attempts for this job.
+     *
+     * @var int
+     */
+    public $tries = 1;
+
+    /**
+     * The user for which to update majors.
+     *
+     * @var \App\User
+     */
+    private $user;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct(User $user)
+    {
+        $this->user = $user;
+        $this->tries = 1;
+        $this->queue = 'buzzapi';
+    }
+
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+        if (null === config('buzzapi.app_password')) {
+            return;
+        }
+
+        if (null === $this->user) {
+            throw new Exception('Attempted to run UpdateMajorsForUser without an existing user');
+        }
+        if ($this->user->is_service_account) {
+            throw new Exception('Attempted to run UpdateMajorsForUser on a service account');
+        }
+
+        $accountResponse = BuzzAPI::select('gtAccountEntitlement')
```

Comment:
This should be a `search`, not a `select`.

Reply:
No, this is correct for the fluent interface. The `select` method will do a `search` in BuzzAPI.

It mostly intentionally resembles SQL.